apps/dashboard/app/javascript/packs/application.js

```diff
@@ -18,7 +18,7 @@ import 'jquery-migrate'
 import 'jquery-ujs'
 
 // Import DataTables
-require('datatables.net-bs/js/dataTables.bootstrap')
+import 'datatables.net-bs/js/dataTables.bootstrap'
```

Comment:
It's better to use `import`

https://insights.untapt.com/webpack-import-require-and-you-3fd7f5ea93c0